### PR TITLE
Add README for OpenJDK Runtime and Development

### PR DESCRIPTION
**Pull request by alexandrevicenzi**
This is related to #993.

### CODE REVIEW

File: src/bci_build/package/openjdk-devel/README.md.j2 [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+To compile the application, without running it inside a container instance, use the following command:
+
+```ShellSession
+$ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} javac Hello.java
+```
+
+## Additional tools
+
+The {{ image.pretty_name }} image includes [Git](https://git-scm.com/) and [Maven](https://maven.apache.org/).
```
Comment (dcermak) [inline src/bci_build/package/openjdk-devel/README.md.j2:52]: I think we can also mention that `ant` is available in the repos.

File: src/bci_build/package/openjdk-devel/README.md.j2 [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+$ podman run -it --rm my-java-app
+```
+
+There are situations, where you don't want to run an application inside a container.
+
+To compile the application, without running it inside a container instance, use the following command:
+
+```ShellSession
+$ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} javac Hello.java
+```
```
Comment (dcermak) [inline src/bci_build/package/openjdk-devel/README.md.j2:48]: I'd suggest to also mention that the image's CMD is the `jshell`.

File: src/bci_build/package/openjdk/README.md.j2 [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# {{ image.pretty_name }} container image
+
+{% include 'badges.j2' %}
+
+## Description
+
+[OpenJDK](https://openjdk.org/) (Open Java Development Kit) is a free and open source implementation of the Java Platform, Standard Edition (Java SE). OpenJDK is the official reference implementation of Java SE since version 7.
+
+The OpenJDK runtime image is intended to be used as a runtime environment. For development, use the OpenJDK development image.
```
Comment (dirkmueller) [inline src/bci_build/package/openjdk/README.md.j2:9]:
```suggestion
The {{ image.pretty_name }} image is intended to be used as a runtime environment. For development, use the corresponding OpenJDK development image.
```

**Review by dirkmueller — APPROVED** [review]
some suggestions inline. I don't like explaining something with the same term, so 

"For development use the development image" is not helpful imho. made a suggestion inline.

**Review by dmpop — COMMENTED** [review]
LGTM

**Pull request merged**